sys/src/9/amd64/devacpi.c

```diff
@@ -2406,6 +2410,20 @@ static char *raw(Atable *atbl, char *start, char *end, void *unused_arg)
 	return start + len;
 }
 
+void outofyourelement(void)
```

Comment:
Not just that, but we need a symbol less-likely to collide with something else. I tried making it static and the compiler just optimizes it away and complains that the called functions aren't used; exactly what I was trying to avoid. So I had to make it extern. :-(